app/src/pages/playground/constants.tsx

```diff
@@ -35,3 +37,17 @@ export const modelProviderToModelPrefixMap: Record<ModelProvider, string[]> = {
   ANTHROPIC: ["claude"],
   OPENAI: ["gpt", "o1"],
 };
+
+export const TOOL_CHOICE_PARAM_CANONICAL_NAME: Extract<
+  CanonicalParameterName,
+  "TOOL_CHOICE"
+> = "TOOL_CHOICE";
+
+export const TOOL_CHOICE_PARAM_NAME = "tool_choice";
```

Comment:
this is not necessarily stable, we need to pull from the backend and map this and use the real models tool_choice name at run time, or allow passing in one of the canonical or param name to the subscription cc @cephalization @anticorrelator, right now they are all the same so it doesn't matter